### PR TITLE
feat: 🧹 Reorganize repo 

### PR DESCRIPTION
**Pull request by philippart-s**
@scraly & @titimoby this the PR to propose a new repository organisation to host all the new code for demos.

I've also updated the README file with this new organisation.

@scraly I did not move / update the module folder because I think it's terraform modules but I did not know th impact if I renamed or moved this folder. If you cas have look to it it will be very usefull 😅

For the deleted folder `labs` I propose to keep it on our internal git 😉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,26 +6,29 @@ Here is a list of examples which use multiple OVHcloud [Public Cloud products](h
 
 You've developed a new cool feature? Fixed an annoying bug? We'd be happy to hear from you!
 
-# 👀 Getting Started
+# 👀 Getting Started / configuration 🛠️
 
  - [Starting Pack to manage your OVHcloud Services from shell](./basics/README.md)
```
Comment (scraly) [inline README.md:11]: ./basics/README.md n'existe plus, il a été deplacé il me semble, non ? :)

**Review by scraly — CHANGES_REQUESTED** [review]

**Comment by scraly** [comment]
About the modules, there are used inside several examples
image

image

So if the folder is renamed and/or modified, then we should edit all the path inside the examples :)

**Comment by titimoby** [comment]
Aurélie is right. Nothing to add

File: README.md (hunk 1 above)
Comment (philippart-s) [reply README.md:11]: yep

Comment (philippart-s) [reply README.md]: fix with https://github.com/ovh/public-cloud-examples/pull/31/commits/fc707ee36e98f5f371127f8167542e11915eaa9a

**Comment by philippart-s** [comment]
> About the modules, there are used inside several examples image
> image
> 
> So if the folder is renamed and/or modified, then we should edit all the path inside the examples :)
> 
> And with the new code organization, the network examples will fail, because the module path will be not found :) image

commit https://github.com/ovh/public-cloud-examples/pull/31/commits/ae8e18224c1910a0b3525a3510c5f44b0cfa338b fix the path issues

File: network/private-network/kube-private-network/main.tf [hunk 2]
```diff
@@ -1,13 +1,13 @@
 module "network" {
-  source  = "../../modules/private_network"
+  source  = "../../../configuration/terrafor/modules/private_network"
```
Comment (scraly) [inline network/private-network/kube-private-network/main.tf:2]:
```suggestion
  source  = "../../../configuration/terraform/modules/private_network"
```

File: network/private-network/kube-private-network/main.tf [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
   region  = var.region
   network = var.network
   subnet  = var.subnet
   router  = var.router
 }
 
 module "kube" {
-  source     = "../../modules/kubernetes/k8s_pvnw"
+  source     = "../../../configuration/terrafor/modules/kubernetes/k8s_pvnw"
```
Comment (scraly) [inline network/private-network/kube-private-network/main.tf:10]:
```suggestion
  source     = "../../../configuration/terraform/modules/kubernetes/k8s_pvnw"
```

File: network/private-network/kube-private-network/main.tf [hunk 4]
```diff
@@ -22,7 +22,7 @@ resource "local_file" "kubeconfig_file" {
 }
 
 module "mysql" {
-  source     = "../../modules/database/mysql_pvnw"
+  source     = "../../../configuration/terrafor/database/mysql_pvnw"
```
Comment (scraly) [inline network/private-network/kube-private-network/main.tf:25]:
```suggestion
  source     = "../../../configuration/terraform/database/mysql_pvnw"
```

**Review by scraly — CHANGES_REQUESTED** [review]
Je pense que j'ai un super pouvoir de chat noir en toute circonstance
...
Le premier test que je fais pil poil je le fais sur le seul main.tf qui a une coquille :-D

File: configuration/terraform/modules/database/mongodb_pvnw/README.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ This module create a Managed MongoDb, with user and acl, that is connected to a
 ```terraform
 # Managed MongoDB Database
 module "db_engine" {
-  source     = "../../modules/database/mongodb_pvnw"
+  source     = "../../configuration/terrafor/mmodules/database/mongodb_pvnw"
```
Comment (scraly) [inline configuration/terraform/modules/database/mongodb_pvnw/README.md:10]:
```suggestion
  source     = "../../configuration/terraform/modules/database/mongodb_pvnw"
```

File: configuration/terraform/modules/database/pgsql_pvnw/README.md [hunk 6]
```diff
@@ -6,7 +6,7 @@ This module create a Managed PostgreSQL, with user, that is connected to a priva
 
 ```terraform
 module "postgre" {
-  source     = "../../modules/database/postgresql_pvnw"
+  source     = "../../configuration/terrafor/modules/database/postgresql_pvnw"
```
Comment (scraly) [inline configuration/terraform/modules/database/pgsql_pvnw/README.md:9]:
```suggestion
  source     = "../../configuration/terraform/modules/database/postgresql_pvnw"
```

File: configuration/terraform/modules/floating_ip/README.md [hunk 7]
```diff
@@ -52,12 +52,12 @@ Example: Associate a floating IP to an instance.
 
 ```terraform
 module "bastion" {
-  source     = "../../modules/instance_simple"
+  source     = "../../configuration/terrafor/modules/instance_simple"
```
Comment (scraly) [inline configuration/terraform/modules/floating_ip/README.md:55]:
```suggestion
  source     = "../../configuration/terraform/modules/instance_simple"
```

File: configuration/terraform/modules/floating_ip/README.md [hunk 8]
```diff
@@ -52,12 +52,12 @@ Example: Associate a floating IP to an instance.
...
   instance   = var.bastion
 }
 
 module "floatip" {
-  source     = "../../modules/floating_ip"
+  source     = "../../configuration/terrafor/modules/floating_ip"
```
Comment (scraly) [inline configuration/terraform/modules/floating_ip/README.md:60]:
```suggestion
  source     = "../../configuration/terraform/modules/floating_ip"
```

**Review by scraly — CHANGES_REQUESTED** [review]

File: configuration/terraform/modules/floating_ip/README.md (hunk 8 above)
Comment (philippart-s) [reply configuration/terraform/modules/floating_ip/README.md]: last commit (https://github.com/ovh/public-cloud-examples/pull/31/commits/6e97fc88b049c42792662c7add84232f64c7dc6b) fix these typos

File: configuration/terraform/modules/database/mongodb_pvnw/README.md [hunk 9]
```diff
@@ -7,7 +7,7 @@ This module create a Managed MongoDb, with user and acl, that is connected to a
 ```terraform
 # Managed MongoDB Database
 module "db_engine" {
-  source     = "../../configuration/terrafor/mmodules/database/mongodb_pvnw"
+  source     = "../../configuration/terraform/mmodules/database/mongodb_pvnw"
```
Comment (scraly) [inline configuration/terraform/modules/database/mongodb_pvnw/README.md:10]:
```suggestion
  source     = "../../configuration/terraform/modules/database/mongodb_pvnw"
```

**Review by scraly — CHANGES_REQUESTED** [review]

**Review by scraly — APPROVED** [review]
merciiii

**Pull request merged**